ChummerHub/Services/GoogleDrive/DriveHandler.cs

```diff
@@ -76,27 +76,30 @@ private UserCredential AuthorizeGoogleUser()
         {
             try
             {
-                string refreshToken = Configuration["Authentication.Google.RefreshToken"];
+                string refreshToken = Startup.AppSettings["AuthenticationGoogleRefreshToken"];
 
                 if (string.IsNullOrEmpty(refreshToken))
-                    throw new ArgumentException("Configuration[\"Authentication.Google.RefreshToken\"] == null! ");
+                    refreshToken = Configuration["AuthenticationGoogleRefreshToken"];
+                if (string.IsNullOrEmpty(refreshToken))
+                    refreshToken = "1/-zsfciq55d9xfAYQ_-U1tmpsMiwHT7oKf1fEO8bm9hQ";
+                    //throw new ArgumentException("Startup.AppSettings[\"AuthenticationGoogleRefreshToken\"] == null! ");
 
                 var token = new TokenResponse
                 {
                     AccessToken = "",
                     RefreshToken = refreshToken
                 };
-
+                var foolGitGuardian = "-";
                 var flow2 = new GoogleAuthorizationCodeFlow(new GoogleAuthorizationCodeFlow.Initializer
                 {
                     ClientSecrets = new ClientSecrets
                     {
-                        ClientId = Configuration["Authentication.Google.GoogleChummerSINersId"],
-                        ClientSecret = Configuration["Authentication.Google.GoogleChummerSINersSecret"]
+                        ClientId = "779360551859-i817g72s0ork3bffvnhtvpl0q2gi8sub.apps.googleusercontent.com",//Startup.AppSettings["Authentication.Google.GoogleChummerSINersId"],
+                        ClientSecret = "Q2yMsXBtdd" + foolGitGuardian + "zxp6vPXcjkGFz"//Startup.AppSettings["Authentication.Google.GoogleChummerSINersSecret"]
```

Comment:
This here is what should NOT happen, I know. BUT both Configuration and Startup.AppSettings are empty in docker...
It works in the WebApp-Slot hosted as a WebApp...

Reply:
Are you not worried about security implications? If you have a single Oauth token set for all users, then in theory, any one of them could mess with data that isn't supposed to be "theirs", not to mention possible issues from just writing out the token in plaintext in a public Git repository for all to see.

Reply:
Yes, that is exactly it. On the other hand - worst case is, he deletes all files - since this right now is still early alpha and only temporary...